### DISCUSSION TITLE
Shared views [Jul 17]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6080] [Discord Threads]
> 🙋‍♂️ **alex.pala**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Shared views are read-only as of now  
#  
> 🙋‍♂️ **alex.pala**  
#### It's not what I'm looking for, people who have to open the link must be able to insert tuples, modify and delete them but only and only those created by him (they must not see those of others, and if they have never added tuples, they must see the empty table )  
#  
> 🙋‍♂️ **alex.pala**  
#### Is possible?  
#  
> 👨‍🦲 **dstala**  
#### Role/User based access control - feature list in our roadmap, currently not supported  
#  
> 🙋‍♂️ **compasssites**  
#### Even simple shared list can have a toggle to make them either read only or editable. There would be many scenarios where we would want someone to fill in the data only in that view and not access the whole base.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1106196838035497012)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
